solidity/random-beacon/package.json

```diff
@@ -19,7 +19,7 @@
     "lint:config:fix": "prettier --write '**/*.@(json|yaml)'"
   },
   "dependencies": {
-    "@keep-network/sortition-pools": "1.2.0-dev.23",
+    "@keep-network/sortition-pools": "github:keep-network/sortition-pools#ad5e3ec",
```

Comment:
https://www.npmjs.com/package/@keep-network/sortition-pools/v/1.2.0-dev.24

is published; I can tweak it and update yarn lock, one sec